src/lightning/__setup__.py

```diff
@@ -48,8 +48,7 @@ def _prepare_extras() -> Dict[str, Any]:
     extras["app-extra"] = extras["app-app"] + extras["app-cloud"] + extras["app-ui"] + extras["app-components"]
     extras["app-all"] = extras["app-extra"]
     extras["app-dev"] = extras["app-all"] + extras["app-test"]
-    extras["data-all"] = extras["data-data"] + extras["data-cloud"] + extras["data-examples"]
-    extras["data-dev"] = extras["data-all"] + extras["data-test"]
+    extras["data-all"] = extras["data-data"]
```

Comment:
I don't see a point in keeping this?

```suggestion
```

Reply:
@carmocca We still want `pip install -e '.[data]'` to work.

Reply:
As `lightning.data` is still available.

Reply:
Ok, removed it.